src/components/NavBar.css

```diff
@@ -0,0 +1,33 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: goldenrod;
+  padding: 0 15px;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.navbar {
+  display: flex;
+}
+
+.navbar li {
+  margin-left: 30px;
+  list-style: none;
+  padding: 5px;
+  font-weight: 500;
+}
+
+.navbar a {
+  text-decoration: none;
+
+  /* color: black; */
```

Comment:
- [ ] To make your code clean, kindly remove, unused commented codes from your file!